package.json

```diff
@@ -46,7 +45,6 @@
     "react-router-dom": "^4.3.1",
     "react-select": "~1.2.1",
     "redux": "^4.0.0",
-    "redux-devtools-extension": "~2.13.5",
```

Comment:
Please keep this one in there, this extension is quite usefull.

Reply:
No, I want to remove the extension because we should not depend on any browser extensions.

The functionality itself should not be affected, the extesion code was only used for type info.

Reply:
In that case i would maybe consider some logger middleware at least for dev env (https://github.com/evgenyrodionov/redux-logger). It can save a lot of time while debugging.

Reply:
@himdel oh yeah sure, great point. No need to have it in deps. :+1: